hudi-common/pom.xml

```diff
@@ -144,6 +144,7 @@
           <artifactId>*</artifactId>
         </exclusion>
       </exclusions>
+      <scope>provided</scope>
```

Comment:
I guess this should be ok. @n3nash agree?

Reply:
I think so. We explicitly whitelist in bundles anwyay, so does not matter